test/integration/targets/lightsail/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+
+- module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key | default(omit) }}'
+      aws_secret_key: '{{ aws_secret_key | default(omit) }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region | default(omit) }}'
+
+- name: Integration test for lightsail module
```

Comment:
If you look at the details of the CI failure, you have a YAML syntax failure. This line is not necessary.
```suggestion
```